Test/Hydrators/DogHydrator.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
```

Comment:
it is impossible to UNIT test a hydrator. this is an integration test as it tests the DB as well. delete this file

Reply:
(liliyia)

Reply:
done